### DISCUSSION TITLE
Qiitaコードブロック内でコメントとして#を入れると見出しになってしまうの防ぎたい

### DISCUSSION THREAD

**Discussion by emi-ki** [increments/qiita-discussions#202] [要望・アイデア・不具合]
### 要望
Qiitaコードブロック内でコメントとして#を入れると見出しになってしまうので、コードブロック内の#は見出しにならないように改善いただくことは可能でしょうか。

![image.png](https://qiita-image-store.s3.ap-northeast-1.amazonaws.com/0/1723719/09aeb105-70a0-8691-4a2b-306513cfa6a5.png)

例えば以下
```py
# Hello World!!を表示する関数
def hello_world():
    return 'Hello World!!'
    
# hello_world()関数を変数に代入する
greeting = hello_world()
print(greeting)
```
コード内でコメントとした記載した「# Hello World!!を表示する関数」「# hello_world()関数を変数に代入する」が見出しに表示されてしまっています。

**Comment by kyntk** [comment]
@emi-ki ご報告いただきありがとうございます！

バグの原因としては、こちらにいただいているDiscussionと同じ内容となっており、今後修正予定です！
https://github.com/increments/qiita-discussions/discussions/178

ただ、不具合修正には時間がかかり、すぐに解決！とはできなそうで、お時間をいただきたいです 🙇🏽

それまではお手数ですが、見出しがおかしいなと思ったら、上記リンクのDiscussionを参考にCode blockの前後に空行をいれていただければ解消すると思いますので、こちらの対応をいただけますでしょうか 🙇🏽

**Reply by emi-ki** [reply]
Code blockの前後に空行を入れたら綺麗に表示されました！
ありがとうございます。

**Reply by kyntk** [reply]
@emi-ki 
目次の不具合修正が完了いたしました！
お待たせして申し訳ございませんでした 🙇

今後は空行を意識せずに目次が生成されるようになります
